packages/atomic/stencil.config.ts

```diff
@@ -16,69 +15,19 @@ import html from 'rollup-plugin-html';
 import {inlineSvg} from 'stencil-inline-svg';
 import tailwind from 'tailwindcss';
 import tailwindNesting from 'tailwindcss/nesting';
-import headlessJson from '../../packages/headless/package.json';
+import {generateExternalPackageMappings} from './scripts/externalPackageMappings';
 import {generateAngularModuleDefinition as angularModule} from './stencil-plugin/atomic-angular-module';
 
 const isProduction = process.env.BUILD === 'production';
 const isCDN = process.env.DEPLOYMENT_ENVIRONMENT === 'CDN';
 
-let headlessVersion: string = '';
-if (isCDN) {
-  console.log('Building for CDN');
-  headlessVersion = 'v' + headlessJson.version;
-}
-
-const packageMappings: {[key: string]: {devWatch: string; cdn: string}} = {
-  '@coveo/headless/commerce': {
-    devWatch: path.resolve(
-      __dirname,
-      './src/external-builds/commerce/headless.esm.js'
-    ),
-    cdn: `/headless/${headlessVersion}/commerce/headless.esm.js`,
-  },
-  '@coveo/headless/insight': {
-    devWatch: path.resolve(
-      __dirname,
-      './src/external-builds/insight/headless.esm.js'
-    ),
-    cdn: `/headless/${headlessVersion}/insight/headless.esm.js`,
-  },
-  '@coveo/headless/product-recommendation': {
-    devWatch: path.resolve(
-      __dirname,
-      './src/external-builds/product-recommendation/headless.esm.js'
-    ),
-    cdn: `/headless/${headlessVersion}/product-recommendation/headless.esm.js`,
-  },
-  '@coveo/headless/recommendation': {
-    devWatch: path.resolve(
-      __dirname,
-      './src/external-builds/recommendation/headless.esm.js'
-    ),
-    cdn: `/headless/${headlessVersion}/recommendation/headless.esm.js`,
-  },
-  '@coveo/headless/case-assist': {
-    devWatch: path.resolve(
-      __dirname,
-      './src/external-builds/case-assist/headless.esm.js'
-    ),
-    cdn: `/headless/${headlessVersion}/case-assist/headless.esm.js`,
-  },
-  '@coveo/headless': {
-    devWatch: path.resolve(__dirname, './src/external-builds/headless.esm.js'),
-    cdn: `/headless/${headlessVersion}/headless.esm.js`,
-  },
-  /*   '@coveo/bueno': {
-    devWatch: path.resolve(__dirname, './src/external-builds/bueno.esm.js'),
-    cdn: `/bueno/${buenoVersion}/bueno.esm.js`,
-  }, */
-};
-
 function generateAliasEntries() {
-  return Object.entries(packageMappings).map(([find, paths]) => ({
-    find,
-    replacement: paths.devWatch,
-  }));
+  return Object.entries(generateExternalPackageMappings(__dirname)).map(
```

Comment:
In this file, I suggest assigning the result of `generateExternalPackageMappings(__dirname)` to a variable and referencing it here and in lines 217 and 223